otter/grade/utils.py

```diff
@@ -9,6 +9,13 @@
 
 OTTER_DOCKER_IMAGE_NAME = "otter-grade"
 
+POINTS_POSSIBLE_LABEL = "points-per-question"
+
+KEY_SCORES_DICT_FILE = "file"
+
+KEY_SCORES_DICT_TTL_PTS = "total_points_earned"
+
+KEY_SCORES_DICT_PCT_CRT = "percent_correct"
```

Comment:
Please rename the `KEY_SCORES_DICT_*` vars to something more readable, e.g. `KEY_SCORES_DICT_FILE` -> `SCORES_DICT_FILE_KEY`, `KEY_SCORES_DICT_PCT_CRT` -> `SCORES_DICT_PERCENT_CORRECT_KEY`. Not concerned about length of the var names but the abbreviations are somewhat opaque.